.circleci/config.yml

```diff
@@ -47,16 +47,24 @@ jobs:
           root: .
           paths:
             - .
-  lint_backend_and_frontend:
+  lint_backend:
     executor: docker-executor
     steps:
       - attach_workspace:
           at: .
       - run:
-          name: Lint backend and frontend
-          command: yarn run lint:ci && cd frontend && yarn run lint:ci
+          name: Lint backend
+          command: yarn run lint:ci
       - store_artifacts:
           path: reports
+  lint_frontend:
+    executor: docker-executor
+    steps:
+      - attach_workspace:
+          at: .
+      - run:
+          name: Lint frontend
+          command: cd frontend && yarn run lint:ci
```

Comment:
```suggestion
```
Get rid of this whole new job block, you don't need it.

Reply:
Yup, @SarahJaine , good point. He meant steps not jobs.